app/ui-utils/client/lib/renderMessageBody.js

```diff
@@ -32,4 +32,4 @@ export const renderMessageBody = mem((message) => {
 
```

Comment:
No PR. Issue in RC posted #14868.  This shall be PR to RC.

Reply:
@vickyokrm I assume the PR #14868 does not really fix the issue properly. There was a reason that you hashed the message object for the cache. Can you please elaborate and potentially open a new PR or at lease create an issue which describes why the current solution is inadequate?